tdvt/tdvt/config_gen/datasource_list.py

```diff
@@ -358,24 +357,27 @@ def load_test(config, test_dir=get_root_dir()):
 class TestRegistry(object):
     """Add a new datasource here and then add it to the appropriate registries below."""
 
-    def __init__(self, ini_file):
+    def __init__(self, ini_file, ds):
         self.dsnames = {}
         self.suite_map = {}
+        self.ds = ds
 
         # Read all the datasource ini files and load the test configuration.
         ini_files = get_all_ini_files_local_first('config')
-        for f in ini_files:
-            logging.debug("Reading ini file [{}]".format(f))
-            config = configparser.ConfigParser()
-            # Preserve the case of elements.
-            config.optionxform = str
-            try:
-                config.read(f)
-            except configparser.ParsingError as e:
-                logging.debug(e)
-                continue
-
-            self.add_test(load_test(config))
+        for d in ds:
```

Comment:
We need to extract this ini parsing logic into a method and run it after `self.suite_map` is set (I believe in the `load_registry()` method below. You could set a breakpoint at tdvt.py:728 to drop into it from the top.

When set, `suite_map` is a dictionary with k:v pairs that look like:
```python
{
    'all': ['athena', 'aurora', ...],
    'standard': [blah, blah, blah, …],
}
```

Right now if we do `tdvt run standard` it won't parse any ini files because it doesn't know which ones to parse and add.

Also, we want to think about more precise pattern matching because there could be shared strings in ds names (ex: postgres and postgres jdbc, or I guess a worse case would be something like mysql and mysql_broken_dont_use 🤣). Not the end of the world, but we might be able to do a straight `==` after splitting the file name rather than an `in` on the filename.